libraries/Microsoft.Bot.PublishValidation/Helpers/ConfigurationOptions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Bot.PublishValidation
+{
+    public class ConfigurationOptions
+    {
+        public bool ForbidSpacesInProjectName { get; set; }
+        public bool RequireBotFile { get; set; }
+        public string RequiredEndpoints { get; set; }
+        public string ForbiddenEndpoints { get; set; }
+        public bool RequireLuisKey { get; set; }
+        public bool RequireQnAMakerKey { get; set; }
+
+        public ConfigurationOptions(string ForbidSpacesInProjectName, string RequireBotFile,
```

Comment:
Add an empty() constructor.

Reply:
I would remove this ctor with all the arguments, and just have a static method do this.